src/main/java/frc/robot/commands/drive/DriveRobotOriented.java

```diff
@@ -16,14 +16,22 @@
 import frc.robot.Constants;
 import frc.robot.RobotContainer;
 import frc.robot.subsystems.Drive;
+import java.util.function.Supplier;
 
 /** Drives the robot in robot-oriented mode. Default command for {@link Drive} subsystem. */
 public class DriveRobotOriented extends Command {
   private final Drive drive;
+  private final Supplier<Double> xScaledSupplier, yScaledSupplier, zScaledSupplier;
```

Comment:
Rename these to xSupplier, ySupplier, etc. In this context we don't care that the controller axes are scaled.